src/Provider/Resource/AbstractStripeResource.php

```diff
@@ -51,7 +51,7 @@ public function fetch(Connector $connector, EncapsulatedOptions $options = null)
                 throw new StripePaymentException(
                     $errorBody['message'],
                     $errorBody['type'],
-                    $errorBody['param'],
+                    isset($errorBody['param']) ?: null,
```

Comment:
* This doesn't do what you think it does; it returns `true`, or `null` instead of the value of `$errorBody['param']`. We should probably have a test for this.
* Optional parameters should appear at the end of the parameter list.